html/pfappserver/lib/pfappserver/Form/Field/ProfileFilter.pm

```diff
@@ -70,7 +70,8 @@ sub filter_deflate {
 sub options_type {
     my $self = shift;
     local $_;
-    return map { /([^:]+)$/; { value => $1, label => $self->_localize($_) } } sort @pf::factory::profile::filter::MODULES;
+    return map { /^profile\.filter\.(.*)$/;{ value => $1, label => $self->_localize($_)} }
```

Comment:
These two lines are like reading Chinese.
Can you rework them to separate the statement and add commenting on what it does